src/db_api/object.js

```diff
@@ -0,0 +1,27 @@
+const { v4: uuidv4 } = require('uuid');
+
+class Order {
+    constructor(group, orderID) {
+        this.belongTo = group;
+        this.orderID  = orderID;
+        this.status   = 0; // 0: unpaid, 1: paid
+        this.store    = "";
```

Comment:
Codacy has a fix for the issue: [Replace `····this.store····=·""` with `this.store·=·''`](https://app.codacy.com/gh/Split-Wise/Uber-Eat-Bot/pullRequest?prid=8101437)
```suggestion
    this.store = '';
```